tools/org.eclipse.n4js.hlc.base/src/org/eclipse/n4js/hlc/base/ErrorExitCode.java

```diff
@@ -42,7 +42,10 @@
 	/** Exit with 10, if required tester could not be loaded. */
 	EXITCODE_TESTER_NOT_FOUND(10, "required tester could not be loaded"),
 	/** Exit with 10, if required tester could not be loaded. */
-	EXITCODE_DEPENDENCY_NOT_FOUND(10, "required dependency is missing");
+	EXITCODE_DEPENDENCY_NOT_FOUND(10, "required dependency is missing"),
+	/** Exit with 11, if srcFiles are invalid. */
+	EXITCODE_SRCFILES_INVALID(10,
```

Comment:
duplicated code `10`

Reply:
sorry, triplicated, three exit codes with code `10`, and javadoc for two of them is wrong

Reply:
done